frontend/src/metabase/css/core/headings.css

```
@@ -19,6 +19,15 @@ h6,
   margin-bottom: var(--default-header-margin);
 }
 
+/**
+ * Correct the font size on `h1` elements within `section` and
+ * `article` contexts in Chrome, Firefox, and Safari.
+ * https://github.com/necolas/normalize.css/blob/master/normalize.css#L40
```

`master` can change, the file can be moved, etc. etc.
I suggest to use a concrete commit hash in the link so that it's more future-proof.

```suggestion
 * https://github.com/necolas/normalize.css/blob/fc091cc/normalize.css#L40
```

---

check the history, it's not moved during 5 years

---

ah, missed your suggestion, thanks!